modules/virt-virtctl-commands.adoc

```diff
@@ -52,8 +52,14 @@ the specified port of the node.
 |`virtctl console <vmi_name>`
 |Connect to a serial console of a virtual machine instance.
 
-|`virtctl vnc <vmi_name>`
-|Open a VNC connection to a virtual machine instance.
+|`virtctl vnc --kubeconfig=$KUBECONFIG <vmi_name>`
+|Open a VNC connection to a virtual machine instance. Access the graphical console of a virtual machine instance through a Virtual Network Client (VNC) which requires a remote viewer on your local machine.
+
+|`virtctl vnc --kubeconfig=$KUBECONFIG --proxy-only=true <vmi-name>`
+|Display the port number and connect manually to the virtual machine instance by using any viewer through the VNC connection.
+
+|`virtctl vnc --kubeconfig=$KUBECONFIG --port=<port-number> <vmi-name>`
+|Specify a port number to run the proxy on the specified port if that port is available. If a port number is not specified, the proxy runs on a random port.
```

Comment:
Add comma

"Specify a port number to run the proxy on the specified port, if that port is available."

Reply:
Added the comma.